src/index.html

```diff
@@ -5,18 +5,18 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
- [ ] It would be better if we use the [Microverse README](https://github.com/microverseinc/readme-template) template, this way our repository will look more professional and attractive to hiring managers